bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/i18n/LocaleProvider.java

```diff
@@ -25,4 +28,10 @@
      */
     Locale getLocale();
 
+    /**
+     * Provides access to the location of the installation
+     *
+     * @return location of the current installation or null if the location is not set
+     */
+    PointType getLocation();
```

Comment:
> No, definitely not - you would like a TranslationProvider. "i18n" is a wide term, which covers locale (and possibly location). So having it in place, but ONLY providing localised (!) texts, is weird.

I agree. `TranslationProvider` should be the name of the current `I18nProvider`.
But I think renaming the `i18nProvider` interface name should not be done in this PR at all.

> So the decision had already been taken a long time ago when the I18nProvider has been introduced

At that time there has been no location and no locale provider at all.

So, I don't see the real "argument" here. It is the same (but I don't think it is one) argument as:
There is currently a `I18nProvider` and a `LocaleProvider`, so the decision to keep them separated has been already taken.